flycheck.el

```diff
@@ -7016,6 +7017,39 @@ See URL `http://hadolint.lukasmartinelli.ch/'."
      (flycheck-remove-error-file-names "/dev/stdin" errors)))
   :modes dockerfile-mode)
 
+(defun flycheck-credo--working-directory (&rest _ignored)
+  "Find directory with from which we can run credo."
+  (and buffer-file-name
+       (locate-dominating-file buffer-file-name "deps/credo")))
+
+(flycheck-def-option-var flycheck-elixir-credo-strict nil elixir-credo
+  "Enable strict mode in credo.
+When non-nil, pass the `--strict' flag to credo."
+  :type 'boolean
+  :safe #'booleanp
+  :package-version '(flycheck . "0.32"))
+
+(flycheck-define-checker elixir-credo
+  "Elixir credo checker."
+  :command ("mix"
+            "credo"
+            (option-flag "--strict" flycheck-elixir-credo-strict)
+            "--format"
+            "flycheck"
+            "--read-from-stdin"
+            source-original)
+  :standard-input t
+  :predicate
+  (lambda () (and buffer-file-name
+                  (locate-dominating-file buffer-file-name "deps/credo")))
+  :working-directory flycheck-credo--working-directory
+  :error-patterns
+  ((info line-start (file-name) ":" line ":" column ": " (or "F" "R" "C")  ": " (message) line-end)
+   (info line-start (file-name) ":" line ": " (or "F" "R" "C")  ": " (message) line-end)
```

Comment:
Rather than two patterns, could you wrap the `":" column` part in an optional group?